### ISSUE TITLE
Slider issues when disabled and when updated

### ISSUE THREAD

**Issue by thbt** [nvaccess/nvda#7072]
NVDA seems to have trouble with elements of `role="slider"`

- When focusing the element, NVDA reads it as `slider editable blank`
- When passing through, it reads it correctly (i.e. `slider editable 8 out of 10`)
- It doesn't speak up changes to `aria-valuenow`
- When I set `aria-disabled` on a slider, it reads it as `slider unavailable editable`: this seems confusing since if it's unavailable, then it's not supposed to be editable (or I'm missing something).

Here is a plunkr with a working rating slider : http://plnkr.co/edit/zfrgHFCW9meogep1UeaG?p=preview

Thanks a lot for your work

**Comment by jcsteh** [comment]
What browser(s) (including version) did you test this with? Also, can you
please elaborate on what you mean by "passing through"? Thanks.

**Comment by thbt** [comment]
@jcsteh I tested this on Windows 7 using chrome (last version: 57.0.2987.13), by "passing through" I mean I switched from forms mode to usual mode (outside of forms mode I can't interact with the rating)

Do you think this might be related to the markup I'm using ?

**Comment by ehollig** [comment]
@thbt is this still an issue? Using latest alpha and latest Firefox, the slider works correctly in browse and focus modes.

**Comment by thbt** [comment]
@ehollig I'm unable to reproduce as I don't have a windows laptop or NVDA.

**Comment by Adriani90** [comment]
I am testing with NVDA 2019.1 and Firefox 66.0.3 as well as Chrome 73. This issue is not reproducible anymore. NVDA reports the slider correctly without "editable". It also reports selected item.
In forms mode (called focus mode in NVDA) the interaction with the slider is possible. In Brose mode it is not possible because the arrow keys jump between elements and do not enter the elements. This is intended in order to allow quicknavigation e.g. with the key l from list to list or c from combo box to combo box.
I am closing this as works for me.
